modules/scheduler/scheduler/src/test/java/org/motechproject/scheduler/it/MotechSchedulerServiceImplBundleIT.java

```diff
@@ -301,7 +499,7 @@ public void shouldThrowExceptionForInvalidCronExpressionWhenreschedulingJob() th
 
     @Test(expected = IllegalArgumentException.class)
     public void shouldThrowExceptionForNullCronJob() throws Exception {
-        schedulerService.scheduleJob(null);
+        schedulerService.scheduleJob((CronSchedulableJob) null);
```

Comment:
Missing unit tests for the serializer/deserializer (however let's hope we can get rid of them).
Missing tests for the controller.
This new update feature in the scheduler service - it should get tested in ITs.